deployments/helm-chart/templates/controller-hpa.yaml

```diff
@@ -18,20 +18,20 @@ spec:
   minReplicas: {{ .Values.controller.autoscaling.minReplicas }}
   maxReplicas: {{ .Values.controller.autoscaling.maxReplicas }}
   metrics:
-    {{- if .Values.controller.autoscaling.targetCPUUtilizationPercentage }}
+    {{- if .Values.controller.autoscaling.targetMemoryUtilizationPercentage }}
```

Comment:
I'm not really sure what difference it makes to swap the two. Do you have a link to any docs about this?

Reply:
You can see the k8s issue here. https://github.com/kubernetes/kubernetes/issues/74099

The problem is somewhere after the manifest is applied in k8s the order swaps. This causes Argocd to think the manifest is always out of sync. By swapping these Argocd will not think the manifest is out of sync.

Reply:
Yeah I saw that, still seems pretty weird to me 🤷